mlir/include/mlir/Dialect/MIOpen/Generator/Conv2dGenerator.h

```diff
@@ -59,7 +62,7 @@ class Conv2dGenerator {
   const Config &getConfig() const { return config; }
   void setKernelName(std::string newName);
 
-  int getKernelCount() const;
+  std::vector<int> getKernelCount() const;
```

Comment:
This is a major departure from existing interface so I'm adding @sjw36 and @jerryyin to review this PR.

Reply:
@whchung Thanks for including me. I can confirm that this will break the client.